src/poliastro/plotting.py

```diff
@@ -182,3 +199,173 @@ def _generate_vals(self, state):
             nu_vals[nu_invalid] = np.nan
 
         return nu_vals
+
+
+def _generate_label(orbit, label):
```

Comment:
I think this could also be used in the current `OrbitPlotter` class.

Reply:
You are right, but I didn't want to touch a line of `OrbitPlotter` to avoid conflicts. We can of course move it!